engine/language-server/src/test/resources/application-test.conf

```diff
@@ -22,11 +22,15 @@ logging-service {
     io.methvin.watcher = error
   }
   appenders = [
+    {
+      name = "memory"
+      forward-to = console
+    },
     {
       name = "console"
-      pattern = "[%level] [%d{yyyy-MM-ddTHH:mm:ssXXX}] [%logger] %msg%n%nopex"
+      pattern = "[%level] [%d{yyyy-MM-ddTHH:mm:ssXXX}] [%logger] %msg%n"
```

Comment:
This change ensures that we print stacktraces. Since now we only do that on failure this is much more useful.